mtgjson5/price_builder.py

```diff
@@ -159,10 +159,14 @@ def build_today_prices() -> Dict[str, Any]:
     tcgplayer = _generate_prices(TCGPlayerProvider())
     card_market = _generate_prices(CardMarketProvider())
     card_kingdom = _generate_prices(CardKingdomProvider())
+    tcgplayer_buylist = _generate_tcgplayer_buylist_prices(TCGPlayerProvider())
 
     final_results = deep_merge_dictionaries(
-        deep_merge_dictionaries(card_hoarder, tcgplayer),
-        deep_merge_dictionaries(card_market, card_kingdom),
+        deep_merge_dictionaries(
```

Comment:
Would like to see DMD rewritten to take any amount of dicts... might ticket this for myself